backend/sftp/sftp.go

```diff
@@ -904,6 +910,7 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 
 	keyFile := env.ShellExpand(opt.KeyFile)
 	pubkeyFile := env.ShellExpand(opt.PubKeyFile)
+	pubKey := env.ShellExpand(opt.PubKey)
```

Comment:
This is not documented above, like for `pubkey_file`, i.e. `+ env.ShellExpandHelp`. But I'm not sure it should be used at all? Expanding a leading `~` into user's home directory path does not make sense here. Or is there a reason I don't see?

Reply:
That is an error on my side. Should be like that.